README.md

```diff
@@ -38,4 +38,14 @@ Some environment variables are used within the project. Find attached the variab
 * start the docker-compose stack with the "development" profile. Set the env-variables (see above). 
 * start the backend - e.g. in IntelliJ or using gradle. Use the spring profile "development"
 * start the frontend - using `npm run serve`. to be safe do an `npm install` and a `./gradlew generateAPIClientFrontend` beforehand.
-* start the E2E cypress tests using `npm run teste2e`
\ No newline at end of file
+* start the E2E cypress tests using `npm run teste2e`
+
+## Dependency Management
+we try to keep our dependencies up to date. Therefore, every two sprints we update dependency versions in a seperate PR.
+To do so:
+* Execute gradlew dependencyUpdates to get a report on Dependencies with updates
+* update `settings.gradle.kts` (for libraries), `build.gradle.kts` (for plugins) and `gradle.properties` (for jacoco)
+* update the gradle wrapper: execute `gradle wrapper --gradle-version X.Y.Z`
+* Do so also for the connected Repos (SkyminderClient, DatalandEDC). Publish new versions of artifacts if required. Use the new artifacts wherever relevant
+* Update Fronted packages: run `npm i npm-check-updates` to install npm-check-updates.
+ Run `npx npm-check-updates -u` to apply updates to the `package.json` file. Run `npm update --save` to `update package-lock.json`
```

Comment:
dies kann man kürzer schreiben ´npx ncu -u´
man sollte noch aufführen das nach einem npm update npm audit fix oft durchzuführen ist
danach sollte man noch testen ob alle funktionen gehen wie zB. die custom npm scripte fake-fixtures

Reply:
* Kurzschreibweise: Ist nicht mehr relevant, weil das Script benutzt wird.
* `npm audit fix`: habe ich eingetragen.
* Testen: Folgt.

Reply:
Testen: Fakefixtures wird jetzt einmal in der CI ausgeführt. Wenn das kaputt geht (also mit non-zero exit code endet) würden wir es merken.